EpsilonDashboard/EpsilonDashboard.cpp

```diff
@@ -15,32 +15,23 @@ EpsilonDashboard::EpsilonDashboard(int& argc, char** argv)
     , businessContainer_(new BusinessContainer(*communicationContainer_, *dataContainer_))
     , presenterContainer_(new PresenterContainer(*dataContainer_))
 {
-    char mode = parseCommandLineArgs();
-    viewContainer_.reset(new ViewContainer(*presenterContainer_, mode));
-}
-
-EpsilonDashboard::~EpsilonDashboard()
-{
-}
-
-char EpsilonDashboard::parseCommandLineArgs()
-{
-    //This function returns 'd' for display mode and 'r' for race mode
-    //The default mode is display mode
     QCommandLineParser parser;
 
     QCommandLineOption raceModeOption("r");
     parser.addOption(raceModeOption);
 
     parser.process(*this);
 
-    char mode = 'd';
+    int mode = DISPLAY;
```

Comment:
Don't pass as ints, pass as the enum type. The whole point of an enum is to keep the name for clarity.